bugbug/utils.py

```diff
@@ -513,3 +513,16 @@ def escape_markdown(text: str) -> str:
         .replace(")", "\\)")
         .replace("|", "\\|")
     )
+
+
+# Remove business days from date range
+
+
+def business_day_range(end_date, start_date):
```

Comment:
Why do we need this logic here? Why `np.busday_count` is not enough?

Reply:
If the final date falls on a weekend, it ignores that day entirely and returns a smaller number than I believe would be intended. I aimed to have it at least add those days to Monday, since that's when work on a bugfix would continue.

Reply:
I think using `np.busday_count()` as is would be OK. 

@marco-c wdyt?

Reply:
@ElusiveEllie Marco agrees. So, please use `np.busday_count()` as is in the function.

Reply:
Understood!